src/exe/udp_client_test/main.cpp

```diff
@@ -0,0 +1,94 @@
+#include <sputter/net/port.h>
+#include <iostream>
+#include <cstring>
+
+using namespace sputter::net;
+
+// Constants for RX port number, TX port number
+// TODO Have me automatically find a free port
+const int RX_PORT = 50000;
+const int TX_PORT = 50001;
+
+// Example of receiving data:
+// std::string address;
+// uint16_t port;
+
+// int bytesReceived = receivingPort.receive(buffer, 256, address, port);
+
+// if (bytesReceived > 0) {
+//   std::cout << "Received " << bytesReceived << " bytes from " << address << ":" << port << std::endl;
+//   std::cout << "Data: " << buffer << std::endl;
+// }
+
+
+// Client code
+int main(int argc, char* argv[]) {
+  // Parse arguments for remote host
+  char* REMOTE_HOST;
+  if (argc > 1) {
+    REMOTE_HOST = argv[1];
+  } else {
+    std::cout << "Usage: " << argv[0] << " [remote host]" << std::endl;
```

Comment:
I wanna eventually get a unified logging system in place that isn't glog, but in the interim if we use glog for everything it'll make it easier to find/replace later.

If you grep around for "LOG" you'll find plenty of examples-- and then you just need to `#include <sputter/system/system.h>`

Reply:
236f023